lib/config/config.go

```diff
@@ -9,29 +9,51 @@ import (
 )
 
 
+func CompileAncestorBranches(branchName string) (result []string) {
```

Comment:
Does this method really "compile" the ancestor branches? What about naming this "CalculateAncestorBranches"?